### ISSUE TITLE
Show error lines

### ISSUE THREAD

**Issue by passy** [passy/android-lint-summary#11]
I'd like to have higher verbosity levels for this, like `-vv`.

**Comment by sindresorhus** [comment]
Maybe also show a snippet of the actual code with a pointer to the issue?

I like the output of JSCS:

<img width="771" alt="screen shot 2015-07-19 at 01 48 03" src="https://cloud.githubusercontent.com/assets/170270/8763969/4967541a-2db8-11e5-9d7a-4b88353a6776.png">

**Comment by passy** [comment]
Yes! And the nice thing is, that this is already part of the XML. I think it's kinda silly to put that into the XML, but in this case it's rather handy.

See https://github.com/passy/android-lint-summary/blob/f481eb2d36f4a9075ffeb51f21472ea96aef7083/Tests/fixtures/1/lint-results.xml#L17
